src/category_theory/limits/preserves/shapes/equalizers.lean

```diff
@@ -0,0 +1,104 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import category_theory.limits.shapes.equalizers
+import category_theory.limits.preserves.basic
+
+/-!
+# Preserving equalizers
+
+Constructions to relate the notions of preserving equalizers and reflecting equalizers
+to concrete forks.
+
+In particular, we show that `equalizer_comparison G f` is an isomorphism iff `G` preserves
+the limit of `f`.
+-/
+
+noncomputable theory
+
+universes v u₁ u₂
+
+open category_theory category_theory.category category_theory.limits
+
+variables {C : Type u₁} [category.{v} C]
+variables {D : Type u₂} [category.{v} D]
+variables (G : C ⥤ D)
+
+namespace preserves_equalizer
+
+variables {X Y Z : C} {f g : X ⟶ Y} {h : Z ⟶ X} (w : h ≫ f = h ≫ g)
+
+/--
+The map of a fork is a limit iff the fork consisting of the mapped morphisms is a limit. This
```

Comment:
This docstring (and the name) don't really seem to match the statement. The conclusion doesn't say that something is a limit, right? It only builds a fork.

Reply:
The conclusion is an equivalence of limits! The `: fork` is misleading, it's to help Lean (and the `simp`) figure out what diagram the fork is for, but the statement is actually of the form:
`is_limit _ ≃ is_limit _`, which is what I think the docstring says

Reply:
Oops, I should read more carefully. Sorry.
So then, how about using `is_limit_map_cone_fork_equiv` as name?

Reply:
Changed, thanks!